### ISSUE TITLE
vue/return-in-computed-property - don't throw an error with return inside switch case

### ISSUE THREAD

**Issue by hipkiss91** [vuejs/eslint-plugin-vue#1049]
**Please describe what the rule should do:**
For computed properties allow something like this
```
formattedTimeLabel: function () {
    switch (this.getJourneyType) {
        case 0: // Default on load OR if selected remains unchanged.
            return this.getPresetString('t1');
        case 1:
            return this.getPresetString('t2');
        case 2:
            return this.getPresetString('t3');
    }
},
```

**What category should the rule belong to?**
- [x] Enforces code style

**Provide 2-3 code examples that this rule should warn about:**
The following would error or throw warning.
```vue
formattedTimeLabel: function () {
    switch (this.getJourneyType) {
        case 0: // Default on load OR if selected remains unchanged.
            return this.getPresetString('t1');
        case 1:
            return this.getPresetString('t2');
        case 2:
            this.getPresetString('t3');
    }
},
```

**Comment by ota-meshi** [comment]
Thank you for this issue.

I think you need to implement to return a value in every code path as follows:

```js
formattedTimeLabel: function () {
    switch (this.getJourneyType) {
        case 0: // Default on load OR if selected remains unchanged.
            return this.getPresetString('t1');
        case 1:
            return this.getPresetString('t2');
        case 2:
            return this.getPresetString('t3');
    }
    return otherValue
},
```

```js
formattedTimeLabel: function () {
    switch (this.getJourneyType) {
        case 0: // Default on load OR if selected remains unchanged.
            return this.getPresetString('t1');
        case 1:
            return this.getPresetString('t2');
        default:
            return this.getPresetString('t3');
    }
},
```

The `vue/return-in-computed-property` rule cannot handle switch cases specially, because it is based on the results of ESLint's code path analysis.
https://eslint.org/docs/developer-guide/code-path-analysis
